cmd/update/update.go

```diff
@@ -0,0 +1,65 @@
+package update
+
+//This update command updates to the latest kubescape release.
+//Example:-
+//          kubescape update
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+
+	logger "github.com/kubescape/go-logger"
+	"github.com/kubescape/kubescape/v2/core/cautils"
+	"github.com/spf13/cobra"
+)
+
+func GetUpdateCmd() *cobra.Command {
+	updateCmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update your version",
+		Long:  ``,
+		RunE: func(_ *cobra.Command, args []string) error {
+			//Checking the user's version of kubescape to the latest release
+			if cautils.BuildNumber == cautils.LatestReleaseVersion {
+				//your version == latest version
+					logger.L().Info( ("You are in the latest version"))
+				} else {
+				//execute the install.sh if linux, install.ps1 for windows,.....depending on your OS
+				const OSTYPE string = runtime.GOOS
+				switch OSTYPE{
+				case "linux":
+					cautils.StartSpinner()
+					//run the installation command for linux
+					cmd, err := exec.Command("./install.sh").Output()
```

Comment:
I apologize for the back and forth here.

In second though, this worked for you only because you ran this kubescape binary in the same path as `install.sh` and `install.ps1`.
The files won't necessarily be there.

I think the right approach here is to detect the full path of the current running Kubescape, and call the install script [directly from github](https://raw.githubusercontent.com/kubescape/kubescape/master/install.sh).
You will need to modify the install script so you can specify the path you want to save Kuibescape in this way you can override the current running Kubescape binary.

We will need to think about users who installed kubescape using homebrew tap.

Reply:
Yeah even I had a thought ,you are right I think we need to find the path of kubescape binary and then override its installation by calling it directly from [github](https://raw.githubusercontent.com/kubescape/kubescape/master/install.sh) and then by overriding its path while running this script.

Reply:
> I apologize for the back and forth here.
No problem I'm eager to learn from my mistakes:)